.github/workflows/ci.yml

```diff
@@ -12,28 +12,17 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        node_version: [10, 12]
+        node_version: [14, 16]
```

Comment:
With the previous ones, `yarn` hadd issues when using the `--immutable` flag (see https://github.com/Fitbit/fitbit-sdk-toolchain/runs/4826568791):
```
Run yarn install --immutable
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'worker_threads'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at /home/runner/work/fitbit-sdk-toolchain/fitbit-sdk-toolchain/.yarn/releases/yarn-3.1.1.cjs:426:2642
    at Object.<anonymous> (/home/runner/work/fitbit-sdk-toolchain/fitbit-sdk-toolchain/.yarn/releases/yarn-3.1.1.cjs:723:7786)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
Error: Process completed with exit code 1.
```

Reply:
We need to support Node 12 as it's still LTS until April. That said, I think we can take this on master and I'll cut a branch from the older commit if we need a new release before then.

Reply:
I'll restore Node 12 and see if the issue happened only for Node 10